src/scripts.js

```diff
@@ -1,11 +1,12 @@
 let sasjs
+let username
```

Comment:
Do we even need the username and password if the login mechanism is Redirected?

Reply:
No, for redirected-login mechanism.

But, code for getting username/password is there to assist default mechanism.

Reply:
removed these variables.